libraries/Microsoft.Bot.Builder.Core/BotContext.cs

```diff
@@ -10,15 +10,15 @@
 
 namespace Microsoft.Bot.Builder
 {
-    public class BotContext : FlexObject, IBotContext
+    public class BotContext : IBotContext
     {
         private readonly BotAdapter _adapter;
-        private readonly IActivity _request;
+        private readonly Activity _request;
```

Comment:
>private readonly Activity _request; [](start = 8, length = 35)

I've actually been wondering if we shouldn't rename this to activity because in the proactive case the activity here isn't a request.